src/components/molecules/UserProfilePicture/UserProfilePicture.types.ts

```diff
@@ -10,4 +11,6 @@ export interface UserProfilePictureProp {
   setSelectedUserProfile: (user: WithId<User>) => void;
   user: WithId<User>;
   reactionPosition?: "right" | "left" | undefined;
+  showNametags?: boolean;
+  currentVenue?: WithId<AnyVenue>;
```

Comment:
It looks like this param isn't used in the code anymore? I'm guessing this is leftover from before you added `const { currentVenue } = useConnectCurrentVenueNG(venueId);`

```suggestion
```

Reply:
removed line